Cannot find module '@umbraco-cms/backoffice/tiptap'

## What type of issue is it? (Choose one - delete the others)
Wrong documentation

## What article/section is this about?
[rich-text-editor/extensions](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/extensions#adding-a-native-extension)

## Describe the issue

When attempting to add the following code sample:

```js
import { UmbTiptapExtensionApiBase } from '@umbraco-cms/backoffice/tiptap';
import { Highlight } from '@tiptap/extension-highlight';

export default class UmbTiptapHighlightExtensionApi extends UmbTiptapExtensionApiBase {
	getTiptapExtensions = () => [Highlight];
}
```
Cannot find module '@umbraco-cms/backoffice/tiptap' or its corresponding type declarations.

![image](https://github.com/user-attachments/assets/e2a2bae7-3e35-44f7-b6d8-3e61245750ec)

--- a/15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/extensions.md
+++ b/15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/extensions.md
@@ -62,39 +62,37 @@ export default class UmbTiptapToolbarHighlightExtensionApi extends UmbTiptapTool
 }
 ```
 
-Once you have the above code in place, they can be referenced in the [package manifest](../../../../../extending/property-editors/package-manifest.md) file.
+Once you have the above code in place, they can be referenced using a [bundle extension type](../../../../../customizing/extending-overview/extension-types/bundle.md).
 
-{% code title="App_Plugins/MyTiptapExtension/umbraco-package.json" lineNumbers="true" %}
-```json
-{
-    "name": "My Tiptap Extension",
-    "version": "1.0.0",
-    "extensions": [
-        {
-            "type": "tiptapExtension",
-            "alias": "My.Tiptap.Highlight",
-            "name": "My Highlight Tiptap Extension",
-            "api": "/App_Plugins/MyTiptapExtension/highlight.tiptap-api.js",
-            "meta": {
-                "icon": "icon-thumbnail-list",
-                "label": "Highlight",
-                "group": "#tiptap_extGroup_formatting"
-            }
-        },
-        {
-            "type": "tiptapToolbarExtension",
-            "kind": "button",
-            "alias": "My.Tiptap.Toolbar.TaskList",
-            "name": "My Highlight Tiptap Toolbar Extension",
-            "api": "/App_Plugins/MyTiptapExtension/highlight.tiptap-toolbar-api.js",
-            "meta": {
-                "alias": "highlight",
-                "icon": "icon-brush",
-                "label": "Highlight"
-            }
-        }
-    ]
-}
+{% code title="manifests.ts" lineNumbers="true" %}
+```js
+export const manifests: Array<UmbExtensionManifest> = [
+	{
+		type: 'tiptapExtension',
+		kind: 'button',
+        alias: 'My Highlight Tiptap Extension',
+        name: 'My.Tiptap.Highlight',		
+		api: () => import('./highlight.tiptap-api.js'),
+		meta:{			
+			"icon": "icon-thumbnail-list",
+			"label": "Highlight",
+			"group": "#tiptap_extGroup_formatting"
+		}
+	},
+	{
+		type: 'tiptapToolbarExtension',
+		kind: 'button',
+		alias: 'My.Tiptap.Toolbar.TaskList',
+		name: 'My Highlight Tiptap Toolbar Extension',
+		js: () => import('./highlight.tiptap-toolbar-api.js'),
+		meta:{
+
+			"alias": "highlight",
+			"icon": "icon-brush",
+			"label": "Highlight"
+		}
+	}
+]
 ```
 {% endcode %}
 

--- a/15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/extensions.md
+++ b/15/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/extensions.md
@@ -45,7 +45,7 @@ import { UmbTiptapExtensionApiBase } from '@umbraco-cms/backoffice/tiptap';
 import { Highlight } from '@tiptap/extension-highlight';
 
 export default class UmbTiptapHighlightExtensionApi extends UmbTiptapExtensionApiBase {
-	getTiptapExtensions = () => [Highlight];
+    getTiptapExtensions = () => [Highlight];
 }
 ```
 
@@ -67,31 +67,31 @@ Once you have the above code in place, they can be referenced using a [bundle ex
 {% code title="manifests.ts" lineNumbers="true" %}
 ```js
 export const manifests: Array<UmbExtensionManifest> = [
-	{
-		type: 'tiptapExtension',
-		kind: 'button',
+    {
+        type: 'tiptapExtension',
+        kind: 'button',
         alias: 'My Highlight Tiptap Extension',
-        name: 'My.Tiptap.Highlight',		
-		api: () => import('./highlight.tiptap-api.js'),
-		meta:{			
-			"icon": "icon-thumbnail-list",
-			"label": "Highlight",
-			"group": "#tiptap_extGroup_formatting"
-		}
-	},
-	{
-		type: 'tiptapToolbarExtension',
-		kind: 'button',
-		alias: 'My.Tiptap.Toolbar.TaskList',
-		name: 'My Highlight Tiptap Toolbar Extension',
-		js: () => import('./highlight.tiptap-toolbar-api.js'),
-		meta:{
-
-			"alias": "highlight",
-			"icon": "icon-brush",
-			"label": "Highlight"
-		}
-	}
+        name: 'My.Tiptap.Highlight',
+        api: () => import('./highlight.tiptap-api.js'),
+        meta:{
+            icon: "icon-thumbnail-list",
+            label: "Highlight",
+            group: "#tiptap_extGroup_formatting"
+        }
+    },
+    {
+        type: 'tiptapToolbarExtension',
+        kind: 'button',
+        alias: 'My.Tiptap.Toolbar.Highlight',
+        name: 'My Highlight Tiptap Toolbar Extension',
+        js: () => import('./highlight.tiptap-toolbar-api.js'),
+        forExtensions: ["My.Tiptap.Highlight"],
+        meta:{
+            alias: "highlight",
+            icon: "icon-brush",
+            label: "Highlight"
+        }
+    }
 ]
 ```
 {% endcode %}

